.github/workflows/build_deploy_test.yml

```diff
@@ -49,10 +49,10 @@ jobs:
         echo "Detected number of ODIM related processes: " $b
         if [ $b -eq 21 ];
```

Comment:
Total number of processes value has been updated with dell and UR plugins, please update with the latest content from development branch.

Reply:
Updated the contents, Please check now